templates/fonts/swift3.stencil

```diff
@@ -23,7 +23,9 @@
   }
 
   {{accessModifier}} func register() {
+    // swiftlint:disable conditional_returns_on_newline
     guard let url = url else { return }
+    // swiftlint:enable conditional_returns_on_newline
```

Comment:
You could also instead directly use `// swiftlint:disable:this conditional_returns_on_newline` just on the affected line,
or `// swiftlint:disable:next conditional_returns_on_newline` on the previous line. This avoids the need to re-enable it on the next line like here.

Reply:
TIL! This is much neater.